pages/copyright.jsx

```diff
@@ -21,7 +21,7 @@ export default function Copyright() {
                 />
             </Head>
             <Page header="Авторство">
-                <Row xs={{ cols: '1' }} md={{ cols: '3' }}>
+                <Row xs={{ cols: '1' }} md={{ cols: '3' }} className="gy-3">
```

Comment:
Вот так лучше выглядит
```suggestion
                <Row xs={{ cols: '1' }} md={{ cols: '3' }} className="gy-5">
```